hws/src/dr_common.py

```diff
@@ -381,7 +381,11 @@ def call_resource_dump(dev, dev_name, segment, index1, num_of_obj1, num_of_obj2,
 }
 
 
+FW_VERSION_MAJOR_CX6 = 22
 FW_VERSION_MAJOR_CX7 = 28
+
+EXTRA_HW_RES_FW_VERSION_MINOR = "39.0230"
+
```

Comment:
It's better to have related names with the same prefix.
At some point you will have feature X starting with some other version.
Change it to FW_VERSION_MINOR_EXTRA_...
Then you have future-proofing for naming:
 - FW_VERSION_MINOR_X_...
 - FW_VERSION_MINOR_Y_...

Reply:
Done